test/erc_to_erc/foreign_bridge.test.js

```diff
@@ -28,14 +29,16 @@ contract('ForeignBridge_ERC20_to_ERC20', async (accounts) => {
       '0'.should.be.bignumber.equal(await foreignBridge.deployedAtBlock())
       false.should.be.equal(await foreignBridge.isInitialized())
       '0'.should.be.bignumber.equal(await foreignBridge.requiredBlockConfirmations())
-      await foreignBridge.initialize(validatorContract.address, token.address, requireBlockConfirmations);
+      await foreignBridge.initialize(validatorContract.address, token.address, requireBlockConfirmations, gasPrice);
```

Comment:
Does it make sense to test that the bridge will not be initialized if incorrect parameters were sent to `initialize()` method?

Reply:
Done